fusor-ember-cli/app/controllers/hypervisor/discovered-host.js

```diff
@@ -86,12 +86,10 @@ export default Ember.Controller.extend(NeedsDeploymentMixin, {
     return (this.get('cntSelectedHypervisorHosts') === this.get('availableHosts.length'));
   }),
 
-  observeAllChecked: Ember.observer('allChecked', function(row) {
-    // TODO
```

Comment:
`allCheck` is not an attribute so it wasn't doing anything. I'm still not 100% sure why this works, since this code in `tr-hypervisor.js` doesn't depend on `checkAll` or `uncheckAll`

```
  isSelectedAsHypervisor: Ember.computed('selectedIds', 'host.id', function () {
    if (this.get('selectedIds')) {
      return this.get('selectedIds').contains(this.get('host.id'));
    }
  })
```